### PR TITLE
✏ Fix typos in docs and source examples

### PR DESCRIPTION
**Pull request by AdrianDeAnda**
Fixing some typos on `async.md`, `unzip-docs.sh` and `test_multipart_installation.py` that I encountered while checking the source code.

### CODE REVIEW

**Comment by AdrianDeAnda** [comment]
@tiangolo Added no coverage on some [tests](https://github.com/tiangolo/fastapi/pull/2102/commits/f7b75bc7077f12b71d34cc05beb9b303f9d9ca6c) as my typo changes might have triggered the codecov check on files that had not been checked in a while.

**Comment by ycd** [comment]
Great catch. also jfyk last commit is duplicate of #2038.

File: fastapi/concurrency.py [hunk 1]
```diff
@@ -20,15 +20,15 @@ def raiser(*args: Any, **kwargs: Any) -> Any:
 
 try:
     from contextlib import asynccontextmanager  # type: ignore
-except ImportError:
+except ImportError:  # pragma: no cover
```
Comment (tiangolo) [inline fastapi/concurrency.py:23]:
```suggestion
except ImportError:
```

File: fastapi/concurrency.py [hunk 2]
```diff
@@ -20,15 +20,15 @@ def raiser(*args: Any, **kwargs: Any) -> Any:
...
     try:
         from async_generator import asynccontextmanager  # type: ignore
     except ImportError:  # pragma: no cover
         asynccontextmanager = _fake_asynccontextmanager
 
 try:
     from contextlib import AsyncExitStack  # type: ignore
-except ImportError:
+except ImportError:  # pragma: no cover
```
Comment (tiangolo) [inline fastapi/concurrency.py:31]:
```suggestion
except ImportError:
```

**Review by tiangolo — COMMENTED** [review]
Awesome! :mag: :detective: 

These 2 `no cover` are not necessary as Codecov takes the coverage from the tests running in all Python versions.

The only disadvantage is that running locally in a single Python version coverage will never be at 100% for that file, but :shrug:

**Comment by tiangolo** [comment]
Thanks! Good catch! :mag: :coffee: 

And thanks @ycd for the help here! :bow:

**Pull request merged**